notebooks/book2/02/sub_super_gauss_plot.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        plt.scatter(x1, x2, marker=".", color="green", linewidths=0.1)
</pre><p>By seeing latexified figure, <code>linewidths=0.1</code> looks high, since all samples are overlapped with each other, you can decrease it to make prettier. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
> * top and right spines are not removed in the three of the four figures

In the original figure both the top and right spines have ticks, so should I remove the spines completely same as the first one or add ticks?